src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -142,9 +144,11 @@ protected void PowerEvent(final PowerEventType x) {
 
     private void recalculateDisplay() {
         int newState = 0;
+        int newType = 0;
 
         for (int x = 0; x < this.getCellCount(); x++) {
             newState |= (this.getCellStatus(x) << (3 * x));
+            newType |= (this.getCellType(x) << (3 * x));
```

Comment:
Since there are only 3 cell types, 2 bits is enough for storage.
```suggestion
            newType |= (this.getCellType(x) << (2 * x));
```